app/models/collection.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# rubocop:disable Style/NumericPredicate
+class Collection < ApplicationRecord
+  def self.create_defaults
+    return if Collection.count > 0
+    Rails.logger.info "Creating default Collections"
+    Collection.create(title: "Research Data", code: "RD")
+    Collection.create(title: "Princeton Plasma Physics Laboratory ", code: "PPPL")
+    Collection.create(title: "Electronic Theses and Dissertations", code: "ETD")
+    Collection.create(title: "Library Resources", code: "LIB")
```

Comment:
@bess - default collections. Notice that I added `code` field for us to refer to them regardless of the final name that they get. I did not add code to link the user to the proper default collection as part of this PR, though.

Reply:
👍🏻